### ISSUE TITLE
Add line labels in instance editor

### ISSUE THREAD

**Issue by vadi2** [health-validator/Hammer#116]
**Is your feature request related to a problem? Please describe.**
It would be great to see where the errors are just by being able to look at the Instance Editor

**Describe the solution you'd like**
Marker(s) for the line on the right side of the editor - so scrolling the editor will show you where the issues lie.

Hovering over the line marker should explain the issues that are there.

**Describe alternatives you've considered**
An in-line annotation for the issue, so you don't have to hover your mouse over anything?

**Comment by pieter-edelman-nictiz** [comment]
Maybe best to tackle this together with #63?

**Comment by vadi2** [comment]
Hm, what's your reasoning for that?

**Comment by pieter-edelman-nictiz** [comment]
It seems related in my view. These two options together would turn the Instance Editor from a basic text box to an actual editor.

**Comment by vadi2** [comment]
Indeed, but I think they're quite big features on their own - might be better to do one at a time.

**Comment by pieter-edelman-nictiz** [comment]
Yeah, you're right, just wanted to somehow relate these two issues.
